Description: Enable UPnP usage in package and disable TORRENTS for jammy
Author: r4sas <r4sas@i2pmail.org>

Reviewed-By: r4sas <r4sas@i2pmail.org>
Last-Update: 2026-10-19

--- i2pd.orig/Makefile
+++ i2pd/Makefile
@@ -31,9 +31,9 @@ # import source files lists
 include filelist.mk
 
 USE_STATIC      := $(or $(USE_STATIC),no)
-USE_UPNP        := $(or $(USE_UPNP),no)
+USE_UPNP        := $(or $(USE_UPNP),yes)
 DEBUG           := $(or $(DEBUG),yes)
-TORRENTS        := $(or $(TORRENTS),yes)
+TORRENTS        := $(or $(TORRENTS),no)
 
 # for debugging purposes only, when commit hash needed in trunk builds in i2pd version string
 USE_GIT_VERSION := $(or $(USE_GIT_VERSION),no)
